lib/src/utils/widgets/linked_in_buttons.dart

```diff
@@ -14,6 +14,7 @@ class LinkedInButtonStandardWidget extends StatelessWidget {
     this.iconAssetPath = 'assets/linked_in_logo.png',
     this.buttonText = 'Sign in with LinkedIn',
     this.buttonColor = Colors.white,
+    this.bgColor = Colors.blue,
```

Comment:
could you please until I am back from vacation, just replace this to `backgroundColor`. Thanks

Reply:
@mdbollman ☝️